ArgoProject/src/systems/CollisionSystem.cpp

```diff
@@ -346,11 +348,11 @@ void app::sys::CollisionSystem::updateCollisionBoxes()
 
 void app::sys::CollisionSystem::playerHazardCollisions()
 {
-	m_registry.view<comp::Collision, comp::Input, comp::Location, comp::Dimensions, comp::Health>(entt::persistent_t())
-		.each([&, this](app::Entity const entity, comp::Collision & collision, comp::Input & input, comp::Location & location, comp::Dimensions & dimensions, comp::Health & health)
+	m_registry.view<comp::Collision, comp::Location, comp::Dimensions, comp::Health>(entt::persistent_t())
```

Comment:
Make sure to call the appropriate `m_registry.prepare<comp::Collision, comp::Location, comp::Dimensions, comp::Health>()` in the system constructor.